mne/tests/test_annotations.py

```diff
@@ -225,14 +224,14 @@ def test_crop_more():
 def test_read_brainstorm_annotations():
     """Test reading for Brainstorm events file."""
     fname = op.join(data_dir, 'events_sample_audvis_raw_bst.mat')
-    annot = read_brainstorm_annotations(fname)
+    annot = read_annotations(fname)
     assert len(annot) == 238
     assert annot.onset.min() > 40  # takes into account first_samp
     assert np.unique(annot.description).size == 5
 
     # Now test with orig_time
     orig_time = np.array([1038942070, 720100], dtype=np.int32)
-    annot = read_brainstorm_annotations(fname, orig_time=orig_time)
+    annot = read_annotations(fname, orig_time=orig_time)
```

Comment:
Here we are testing the `orig_time` for brainstorm but I'm not sure that there's an application for it. @annesodub, @agramfort, @ftadel is it a real use case or is a **YAGNI** code that got in by mistake in #4910?

If the `times[0]` in all `events_*.mat` files refer to `first_samp` as it seems to be the case based on this:
https://github.com/mne-tools/mne-python/blob/f0ef23913f127781d5800e22a6ed8f947d7a6e74/mne/annotations.py#L477

Reply:
It doesn't look very useful, but I didn't write it, I don't know what this is for.

Reply:
If brainstorm always stores the events with respect to fist_sample, then yes it is not useful.

Reply:
Events are stored in time, not in samples wrt the first sample recorded in the data file.
The field "samples" is now redundant with the field "times", it is obtained by multiplying the time with the sampling frequency (the result is rounded), you can ignore it.

The beginning of the file in time is stored separately (time value of the sample #1), in the file link, field sFile.prop.times(1). This value is zero for most of the file formats, except for Neuromag recordings.
https://neuroimage.usc.edu/brainstorm/Tutorials/EventMarkers#On_the_hard_drive